<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>280</height>
           </size>
          </property>
          <property name="title">
           <string>CPUBox</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCustomPlot" name="cpuPlotLayout" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="cpuLabelFrame">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>80</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="NetworkBox">
          <property name="title">
           <string>Network Usage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCustomPlot" name="networkPlot" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>130</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="networkRxLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="networkTxLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="MemoryBox">
          <property name="title">
           <string>Memory Usage</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCustomPlot" name="memoryPlot" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>120</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="memoryUsageLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="DiskBox">
          <property name="title">
           <string>Disk Usage</string>
          </property>
          <widget class="QLabel" name="ioReadLabel">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>671</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
          <widget class="QLabel" name="ioWriteLabel">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>90</y>
             <width>681</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
